VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/waiting-room-shared/waiting-room-base.component.ts

```diff
@@ -1307,6 +1307,10 @@ export abstract class WaitingRoomBaseDirective implements AfterContentChecked {
         this.streamInMain = !this.streamInMain;
     }
 
+    videoClosedExt() {
+        this.logger.debug(`${this.loggerPrefix} - video closed`);
```

Comment:
Did you mean to have a method that just logs a single line?

Reply:
Yep, it's to extend the functionality of the setShowVideo function, so I can override it in the child - judge waiting room - class